From 256e00c5bdf3fb3210292100d181c41c7032d18e Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Fri, 1 Mar 2019 11:06:02 +0200
Subject: [PATCH 018/269] s32g: Clean up secure interrupt list

Remove unnecessarily enabled interrupts in the TF-A.
No need to explicitly clear the SCR_EL3.{FIQ,IRQ} bits at this stage, as
that seems to be taken care explicitly by the gicv3 cpuif initialization
functions.

Issue: ALB-3508

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/s32g/s32g275_bl31.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/plat/s32g/s32g275_bl31.c b/plat/s32g/s32g275_bl31.c
index 4e87c0ff5..5a77eca84 100644
--- a/plat/s32g/s32g275_bl31.c
+++ b/plat/s32g/s32g275_bl31.c
@@ -120,12 +120,7 @@ static unsigned int plat_s32g275_mpidr_to_core_pos(unsigned long mpidr)
 
 static uintptr_t rdistif_base_addrs[PLATFORM_CORE_COUNT];
 
-static const interrupt_prop_t interrupt_props[] = {
-	INTR_PROP_DESC(29, GIC_HIGHEST_SEC_PRIORITY,
-		       INTR_GROUP0, GIC_INTR_CFG_EDGE),
-	INTR_PROP_DESC(153, GIC_HIGHEST_NS_PRIORITY,
-		       INTR_GROUP1S, GIC_INTR_CFG_LEVEL),
-};
+static const interrupt_prop_t interrupt_props[] = {};
 
 const gicv3_driver_data_t s32g275_gic_data = {
 	.gicd_base = PLAT_GICD_BASE,
-- 
2.17.1

